src/main/java/org/jboss/jandex/FieldInfo.java

```diff
@@ -34,7 +34,7 @@
  * @author Jason T. Greene
  *
  */
-public final class FieldInfo implements AnnotationTarget {
+public final class FieldInfo implements AnnotationTarget, Comparable<FieldInfo> {
```

Comment:
I'm surprised you don't have the equivalent for `MethodInfo`?

Reply:
`MethodInfo` is made `Comparable` too in this PR. See below. Or you mean something else?

Reply:
Ah no, silly me, I missed it. Sorry for the noise!